### PR TITLE
DDPB-243 character limiter js module built. limit applied to date fields

### PR DESCRIPTION
**Pull request by Biggsen**

### CODE REVIEW

File: src/AppBundle/Resources/assets/javascripts/modules/characterLimiter.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+    if (typeof GOVUK === 'undefined') {
+        root.GOVUK = {};
+    }
+
+    var limitChars = function (containerSelector) {
+        var limitElement = $(containerSelector).find("[class*='js-limit-chars-']");
+
+        limitElement.on('keyup input paste change', function (event) {
+            // Get the classes
```
Comment (elvisciotti) [inline src/AppBundle/Resources/assets/javascripts/modules/characterLimiter.js]: what about $(event.target) in a variable ?

File: src/AppBundle/Resources/assets/javascripts/modules/characterLimiter.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+            var elClass = $(event.target).attr('class');
+            // Get the limiter value (the 'x')
+            var charsLimit = elClass.substr(elClass.indexOf("limit-chars-")+12,1);
```
Comment (elvisciotti) [inline src/AppBundle/Resources/assets/javascripts/modules/characterLimiter.js]: parseInt ?

File: src/AppBundle/Resources/assets/javascripts/modules/characterLimiter.js (hunk 1 above)
Comment (Biggsen) [inline src/AppBundle/Resources/assets/javascripts/modules/characterLimiter.js]: good one. done

File: src/AppBundle/Resources/assets/javascripts/modules/characterLimiter.js (hunk 2 above)
Comment (Biggsen) [inline src/AppBundle/Resources/assets/javascripts/modules/characterLimiter.js]: Added.

**Pull request merged**